app/views/groups/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= render "partials/navbar" %>
+
+<section>
+    <%@groups.each do |group|%>
+      <div class = "body-flex">
+        <img src=<%=group.icon%> alt="icon">
+
+        <%= link_to group_path(group.id), :remote => true do %>
+          <h3><%=group.name%></h3>
+        <%end%>
+
+        <p><%=group.created_at%></p>
+
+ <h3>$<%=Group.total(group.id)%>.00</h3> 
+
+          <%= button_to "Delete", group_path(group.id), html: {'data-turbo' => "false"}, :method => :delete %>
+      </div>
+    <%end%>
+</section>
```

Comment:
- [ ] Here as well Please fix the image size so that your UI looks great.

![Screenshot from 2022-03-25 03-45-39](https://user-images.githubusercontent.com/22434660/160020163-9aef91f0-032c-495f-9540-c2d91a6c0192.png)